data/mods/Mutant_Power_Armor/armor.json

```diff
@@ -0,0 +1,216 @@
+    "weight": 12214,
+    "volume": 100,
+    "price": 70000,
+    "to_hit": 1,
+    "bashing": 1,
+    "material": [ "steel" ],
+    "symbol": "[",
+    "color": "light_gray",
+    "covers": [ "TORSO", "ARMS", "HANDS", "LEGS", "FEET" ],
+    "coverage": 95,
+    "encumbrance": 50,
+    "storage": 10,
+    "warmth": 50,
+    "material_thickness": 8,
+    "environmental_protection": 6,
+    "flags": [ "WATERPROOF", "STURDY", "OVERSIZE", "ALLOWS_NATURAL_ATTACKS" ]
+  },
+  {
+    "id": "depowered_mutant_helmet",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": "salvaged modified power armor helmet",
+    "description": "A basic helmet, designed for use with the DoubleTech Power Armor, Mk. I.  Offers excellent protection from both attacks and environmental hazards.  This suit has had its internal computer and cooling system stripped out, meaning it no longer requires power, but it has no internal chronometer and doesn't provide internal thermal regulation.",
+    "weight": 2416,
+    "volume": 20,
+    "price": 30000,
+    "to_hit": 1,
+    "bashing": 1,
+    "material": [ "steel" ],
+    "symbol": "[",
+    "color": "light_gray",
+    "covers": [ "HEAD", "EYES", "MOUTH" ],
+    "coverage": 95,
+    "encumbrance": 50,
+    "warmth": 50,
+    "material_thickness": 8,
+    "environmental_protection": 6,
+    "qualities": [ [ "GLARE", 2 ] ],
+    "flags": [ "WATERPROOF", "STURDY", "SUN_GLASSES", "OVERSIZE", "ALLOWS_NATURAL_ATTACKS" ]
+  },
+  {
+    "id": "mutant_power_armor_basic",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": "modified basic power armor",
+    "description": "The DoubleTech Power Armor, Mk. I: A heavy suit of basic power armor, offering very good protection against attacks, but hard to move in.  The UPS compartment can store other things if you don't mind powering the suit yourself.  Like all DoubleTech power armor, the control and climate-regulation systems require direct skin contact.",
+    "weight": 20339,
+    "volume": 100,
+    "price": 7000000,
+    "to_hit": 1,
+    "bashing": 1,
+    "material": [ "steel" ],
+    "symbol": "[",
+    "color": "light_gray",
+    "covers": [ "TORSO", "ARMS", "HANDS", "LEGS", "FEET" ],
+    "coverage": 100,
+    "encumbrance": 60,
+    "storage": 10,
+    "warmth": 90,
+    "power_armor": true,
+    "material_thickness": 11,
+    "environmental_protection": 16,
+    "use_action": { "type": "ups_based_armor", "activate_msg": "Your power armor engages." },
+    "flags": [ "WATERPROOF", "STURDY", "ELECTRIC_IMMUNE", "OVERSIZE", "ALLOWS_NATURAL_ATTACKS" ]
+  },
+  {
+    "id": "mutant_power_armor_frame",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": "modified power armor hauling frame",
+    "description": "A heavy duty hauling frame designed to interface with power armor.",
+    "weight": 1640,
+    "volume": 48,
```

Comment:
```suggestion
    "volume": "12 L",
```

Reply:
thanks a bunch! I was honestly stumped by the end of file linebreak